sdcm/nemesis.py

```diff
@@ -161,6 +161,14 @@ def disrupt_stop_start_scylla_server(self):
         self.target_node.stop_scylla_server(verify_up=False, verify_down=True)
         self.target_node.start_scylla_server(verify_up=True, verify_down=False)
 
+    def disrupt_restart_node(self):
+        self._set_current_disruption('RebootNode %s' % self.target_node)
```

Comment:
RebootNode -> RestartNode

Reply:
Thx, I initially wanted to create reboot node (we will need to do that as well)